api.go

```diff
@@ -270,8 +270,9 @@ type (
 	UploadTaskParams struct {
 		URL        string `json:"url,omitempty"`
 		Encryption struct {
-			Key       string `json:"key"`
-			Algorithm string `json:"algorithm,omitempty"`
+			EncryptedKey string `json:"encryptedKey,omitempty"`
+			Key          string `json:"key,omitempty"`
```

Comment:
Can you add a comment saying why this is still here? We should not have both/never pass unencrypted keys around, so it needs at least a TODO to remove it once we kill the other experiment.

Reply:
To me, it's also not clear why we need both encrypted and decrypted keys here. Looking at the design [VOD Access Control V2](https://www.notion.so/livepeer/VOD-Access-Control-V2-Technical-doc-240d77ebac824db998172c4e00542e4b), I don't see any place where Catalyst would need to fetch encrypted and descrypted keys from Studio. Could you maybe update the design or add some comments here?

Reply:
I think this was a reminiscent of an old experiment I had implemented where the key came unencrypted in this field. I don't think it should stay around, so I'd rather remove it and avoid the confusion in the schema.